manifests/t/TheOSCARTeam/OSCAR/1.3.1.500/TheOSCARTeam.OSCAR.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: TheOSCARTeam.OSCAR
+PackageVersion: 1.3.1.500

```

This must match what is shown in control panel
```suggestion
PackageVersion: 1.3.1-Win64-3ca1ee24
```